modules/java/generator/src/cpp/listconverters.h

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Please use `.hpp` extension (because it is C++ header file).

Also please append OpenCV license header to this file and to `.cpp` file:

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html

// Author: abratchik
```